src/components/AllComments.jsx

```diff
@@ -1,10 +1,28 @@
 import moment from "moment";
 import { useState, useEffect } from "react";
 import { getComments } from "../utils/api";
+import { deleteComment } from "../utils/api";
+import React from "react";
 
 export const AllComments = ({ review }) => {
   const [allComments, setAllComments] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const refreshPage = () => {
+    window.parent.location = window.parent.location.href;
+  };
```

Comment:
This achieves what it says - refreshing the page - but that is probably slightly overkill for what you want to do. It means that your app has to then be fetched from scratch, when in reality all you want to do is re-render a small part of it: the comments section. I would think about 1 of:
1. Re-fetching the comments data with the deleted comment
2. Optimistically removing the comment (this would be quite tricky though)